dotcom-rendering/src/components/Masthead/Titlepiece/ExpandedNav/ExpandedNavColumn.tsx

```diff
@@ -0,0 +1,329 @@
+			display: none;
+		}
+	}
+`;
+
+export const lineStyle = css`
+	${from.desktop} {
+		display: none;
+	}
+	background-color: ${themePalette('--sub-nav-horizontal-section-divider')};
+	content: '';
+	display: block;
+	height: 1px;
+	margin-left: 50px;
+	right: 0;
+`;
+
+const columnStyle = css`
+	${textSans15};
+	list-style: none;
+	/* https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns */
+	/* Needs double escape char: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#es2018_revision_of_illegal_escape_sequences */
+
+	&::before {
+		content: '\\200B'; /* Zero width space */
+		display: block;
+		height: 0;
+		width: 0;
+	}
+
+	margin: 0;
+	padding-bottom: 10px;
+	position: relative;
+
+	${from.desktop} {
+		width: ${pillarWidthsPx.tablet}px;
+		float: left;
+		position: relative;
+		border-left: 1px solid ${themePalette('--masthead-nav-lines')};
+
+		:after {
+			height: 100%;
+			left: 0;
+			width: 1px;
+		}
+
+		:first-of-type {
+			border-left: none;
+		}
+	}
+`;
+
+const columnStyleFromLeftCol = css`
+	${from.leftCol} {
+		width: ${pillarWidthsPx.leftCol}px;
+	}
+	${from.wide} {
+		width: ${pillarWidthsPx.wide}px;
+	}
+`;
+
+export const ExpandedNavColumn = ({
+	column,
+	index,
+	showLineBelow,
+	hasPageSkin,
+}: {
+	column: PillarLinkType | EditionLinkType;
+	index: number;
+	showLineBelow: boolean;
+	hasPageSkin?: boolean;
+}) => {
+	// As the elements are dynamic we need to specify the IDs here
+	//Replace whitespace with hyphen https://stackoverflow.com/questions/3794919/replace-all-spaces-in-a-string-with/3795147#3795147
+	const columnId = column.title.split(' ').join('-');
+	const columnInputId = `${columnId}-checkbox-input`;
+	const collapseColumnInputId = `${columnId}-button`;
+	const ariaControls = `${columnId.toLowerCase()}Links`;
+
+	return (
+		<li
+			css={[
+				columnStyle,
+				!hasPageSkin && columnStyleFromLeftCol,
+				pillarDivider,
+			]}
+			role="none"
+		>
+			{/*
+                IMPORTANT NOTE: Supporting NoJS and accessibility is hard.
+
+                We therefore use JS to make the Nav elements more accessible. We add a
+                keydown `addEventListener` to toggle the column drop down.
+                This is not a perfect solution as not all screen readers support JS
+                https://webaim.org/projects/screenreadersurvey8/#javascript
+            */}
+			<script
+				dangerouslySetInnerHTML={{
+					__html: `document.addEventListener('DOMContentLoaded', function(){
+                        var columnInput = document.getElementById('${collapseColumnInputId}');
+
+						if (!columnInput) return; // Sticky nav replaces the nav so element no longer exists for users in test.
+
+						columnInput.addEventListener('keydown', function(e){
+                            // keyCode: 13 => Enter key | keyCode: 32 => Space key
+                            if (e.keyCode === 13 || e.keyCode === 32) {
```

Comment:
FYI [`keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) in favour of [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)